### PR TITLE
add inithial modules

### PR DESCRIPTION
**Pull request by smdmts**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,38 @@
...
+TODO: Write short description here and build.gradle file.
+
+## Overview
+
+* **Plugin type**: parser
+* **Guess supported**: no
+
+## Configuration
+
+- **option1**: description (integer, required)
```
Comment (civitaspo) [inline README.md]: As far as I can see, there is no option, right?

File: src/main/scala/org/embulk/parser/firebase_avro/FirebaseAvroParserPlugin.scala [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      LoanPattern(new PageBuilder(Exec.getBufferAllocator, schema, output)) {
+        pb =>
+          while (efis.nextFile()) {
+            addRecord(efis, pb)
+          }
+          pb.finish()
+      }
+    }
+
+  def addRecord(is: InputStream, pb: PageBuilder): Unit =
```
Comment (civitaspo) [inline src/main/scala/org/embulk/parser/firebase_avro/FirebaseAvroParserPlugin.scala]: Do you mean `addRecords`?

**Review by civitaspo — COMMENTED** [review]
Just commented..

File: src/main/scala/org/embulk/parser/firebase_avro/FirebaseAvroParserPlugin.scala (hunk 2 above)
Comment (smdmts) [reply src/main/scala/org/embulk/parser/firebase_avro/FirebaseAvroParserPlugin.scala]: Thanks, I'll fix it.

File: README.md (hunk 1 above)
Comment (smdmts) [reply README.md]: Sorry, the document is now writing. This is WIP.

**Comment by smdmts** [comment]
@civitaspo Thanks a lot. I fixed it.

**Pull request merged**